cli/package.json

```diff
@@ -26,7 +26,7 @@
     "clean-build": "rm -rf ./build",
     "compile": "tsc -p .",
     "copy-templates": "bun run copyfiles -u 2 -a \"./src/templates/**/*\" ./build/templates",
-    "format": "eslint \"**/*.{js,jsx,ts,tsx}\" --fix && prettier \"**/*.{js,jsx,ts,tsx,json}\" --write",
+    "format": "eslint \"**/*.{js,jsx,ts,tsx}\" --fix --resolve-plugins-relative-to . && prettier \"**/*.{js,jsx,ts,tsx,json}\" --write",
```

Comment:
why was this added?

Reply:
<img width="1499" alt="image" src="https://github.com/danstepanov/create-expo-stack/assets/30200380/bf1e0375-88e4-4fb9-9b0f-bdb94c1ca7b9">

So running `yarn create-expo-stack` has found multiple prettier instances
So this enables it to choose the one relative to the folder I am in.
This happens to me as I am running this command inside of current cli to do fast iteration of changes.

<img width="759" alt="image" src="https://github.com/danstepanov/create-expo-stack/assets/30200380/eef6406d-f01f-417e-a8db-d433f830f47c">

Reply:
@ernestoresende @sammoore thoughts?

Reply:
@danstepanov This is great!